doc/source/user_guide/groupby.rst

```diff
@@ -1067,7 +1067,7 @@ missing values with the ``ffill()`` method.
    ).set_index("date")
    df_re
 
-   df_re.groupby("group").resample("1D").ffill()
+   df_re.groupby("group")[["val"]].resample("1D").ffill()
```

Comment:
This is different than the original code.

In this example the user probably want to keep "group" column, so I'd do`[["group", "val"]]`.

Reply:
Why would you want to have the group in both the index and the columns?